packages/nextjs/hooks/scaffold-eth/useTransactor.tsx

```diff
@@ -90,8 +90,10 @@ export const useTransactor = (_signer?: Signer): TTransactionFunc => {
       }
       // TODO handle error properly
       console.error("⚡️ ~ file: useTransactor.ts ~ error", error);
-      const message = getParsedEthersError(error);
-      notification.error(message);
+      const err = extractKeyValueErrorPairs(JSON.stringify(error)) as any;
```

Comment:
Why are we stringifying the error here instead of using it directly as an error object? That would avoid the need for the JSON parsing inside, no?

Can't we just improve the parsing done by `getParsedEthersError` rather than recreating something similar?

Reply:
I do not want to temper with what was there since i will be making use of it